### PR TITLE
Add CLI options for history file and pager

### PR DESCRIPTION
**Pull request by nineinchnick**
## Description

Add CLI options for pager and history file, which previously could only be set using environmental variables. Having them available as options allows setting them in the configuration file.

Extracted from #12208 

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

**Comment by nineinchnick** [comment]
@kokosing this is ready to go, PTAL.

File: client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java [hunk 1]
```diff
@@ -148,6 +151,9 @@
     @Option(names = "--output-format", paramLabel = "<format>", defaultValue = "CSV", description = "Output format for batch mode [${COMPLETION-CANDIDATES}] " + DEFAULT_VALUE)
     public OutputFormat outputFormat;
 
+    @Option(names = "--pager", paramLabel = "<pager>", description = "Path to the pager program used to display the query results")
```
Comment (kokosing) [inline client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java:154]: It loosk like `System.getenv("TRINO_PAGER"))` is default, can reference to it?

**Review by kokosing — APPROVED** [review]
% comment

File: client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java (hunk 1 above)
Comment (nineinchnick) [reply client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java]: Done!

**Comment by kokosing** [comment]
The CI is failing. Is it related?

**Comment by nineinchnick** [comment]
> The CI is failing. Is it related?

No, the issues in web ui job should be already fixed on master, I rebased.

**Comment by kokosing** [comment]
> This is not user-visible or docs only and no release notes are required.

I think it is user-visible change and docs are release notes are required. @nineinchnick Please provide content.

**Comment by kokosing** [comment]
Thanks, merged

**Comment by nineinchnick** [comment]
I updated the PR description with suggested release notes.

**Pull request merged**